internal/core/src/storage/Util.cpp

```diff
@@ -31,7 +33,22 @@
 namespace milvus::storage {
 
 std::map<std::string, ChunkManagerType> ChunkManagerType_Map = {
-    {"local", ChunkManagerType::Local}, {"minio", ChunkManagerType::Minio}};
+    {"local", ChunkManagerType::Local},
+    {"minio", ChunkManagerType::Minio},
+    {"remote", ChunkManagerType::RemoteStore}};
```

Comment:
what's mean of remoteStore,  minio is also remote

Reply:
In the future, configuring it as "remote" will be suggested, with "Minio" being just one of the available providers. @zhagnlu @PowderLi